src/sentry/testutils/cases.py

```diff
@@ -57,7 +57,8 @@
 from django.db import DEFAULT_DB_ALIAS, connection, connections
 from django.db.migrations.executor import MigrationExecutor
 from django.http import HttpRequest
-from django.test import TestCase, TransactionTestCase, override_settings
+from django.test import TestCase as DjangoTestCase
```

Comment:
This is necessary since down in the file we declare the class `TestCase`, thus, making inheriting from the `TestCase` class ambiguous.